### PR TITLE
Candidate process and compendium deletion

### PR DESCRIPTION
**Pull request by nuest**

### CODE REVIEW

**Comment by nuest** [comment]
@7048730 @jansule Feedback welcome (regarding clarity, works for client)

File: docs/compendium/candidate.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+After uploading a compendium is _not_ instantly publicly available.
+It is merely a **candidate**, because metadata still must be completed for the compendium to be valid.
+
+The following process models this intermediate state of a compendium.
+
+## Creation and view
+
+Candidates can be identified by the property `candidate`.
+It is set to `true` directly after creating a new compendium by [upload](upload.md) or [public share submission](public_share.md).
```
Comment (jansule) [inline docs/compendium/candidate.md]: @nuest Will the  property be set to `true` in the front-end or in back-end?

Comment (nuest) [reply docs/compendium/candidate.md]: Regular readers won't find that issue, tried to clarify in https://github.com/o2r-project/o2r-web-api/pull/58/commits/e74f35cf66e6e5ea788e21ac536c3f53b100cbdb

**Comment by nuest** [comment]
**Ready to be merged**

**Comment by nuest** [comment]
Includes in `0.12.0`, see https://hub.docker.com/r/o2rproject/o2r-muncher/builds/bkvundvafd9bvket8jy2nb4/

**Pull request merged**